src/ini_get.f90

```diff
@@ -44,18 +44,40 @@ subroutine ini_get (restart, expno_r, restart_date)
 
 !     h and A set to zero on open boundaries
 
-             if (i.eq.0 .or. i.eq.nx+1 .or. j.eq.0 .or. j.eq.ny+1) &
-                  h(i,j) = 0d0
-             if (i.eq.0 .or. i.eq.nx+1 .or. j.eq.0 .or. j.eq.ny+1) &
-                  A(i,j) = 0d0
+             if ((i.eq.0 .or. i.eq.nx+1) .and. Periodic_y .eq. 0) then
+                h(i,j) = 0d0
+                A(i,j) = 0d0   
+             endif
+             if ((j.eq.0 .or. j.eq.ny+1) .and. Periodic_x .eq. 0) then
+                h(i,j) = 0d0
+                A(i,j) = 0d0  
+             endif
 
-             Pp(i,j) = 0d0 
-             P(i,j)  = 0d0
+!     Compression experiment: we set bands of open water at the top and sides
```

Comment:
Uni-axial loading experiment instead of compression

Reply:
```suggestion
!     Uniaxial loading experiment: we set bands of open water at the top and sides
```